hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyRequest.java

```diff
@@ -307,7 +307,6 @@ protected OMClientResponse prepareCreateKeyResponse(@Nonnull KeyArgs keyArgs,
 
 
         if (omAction == OMAction.CREATE_FILE) {
-          ozoneManager.getMetrics().incNumCreateFile();
```

Comment:
Just curious.. why are these metrics updates removed? Were they wrong?

Reply:
Because they are updated at starting on the method in Line 167: OMFileCreateRequest. Similarly for OMKeyCreateRequest also.